python/lbann/contrib/args.py

```diff
@@ -28,9 +29,41 @@ def add_scheduler_arguments(parser):
     parser.add_argument(
         '--account', action='store', type=str,
         help='scheduler account', metavar='NAME')
+    parser.add_argument(
+        '--reservation', action='store', type=str,
+        help='scheduler reservation', metavar='NAME')
     parser.add_argument(
         '--time-limit', action='store', type=int,
         help='time limit (in minutes)', metavar='MIN')
+    parser.add_argument(
+        '--setup-only', action='store_true',
+        help='set up (but do not run) the experiment')
+
+def get_launcher_kwargs(args):
```

Comment:
To make it match `add_scheduler_arguments`, how about:
```suggestion
def get_scheduler_kwargs(args):
```